src/methods.jl

```diff
@@ -332,17 +366,19 @@ function combine(frames::Vararg{<:AbstractArray{<:Number}}; method = median)
     firstframe = first(frames)
     dim = ndims(firstframe) + 1
     shape = size(firstframe)
-    return reshape(method(LazyStack.stack(frames), dims = dim), shape)
+    return reshape(method(LazyStack.stack(frames...), dims = dim), shape)
 end
 
-combine(frames; method = median, kwargs...) = combine(frames...; method = method, kwargs...)
+combine(frames; kwargs...) = combine(frames...; kwargs...)
```

Comment:
I think this creates ambiguities if I were to call `combine(randn(512, 512))` ie combine one frame. I think this behavior should be allowed- if it doesn't work as is you'll just have to play around with typing the methods.

Reply:
This one is not ambiguous:
```
julia> @which combine(randn(512, 512))
combine(frames::AbstractArray{#s27,N} where N where #s27<:Number...; method) in CCDReduction at /home/siddharth/.julia/dev/CCDReduction.jl/src/methods.jl:366
```
Goes to the `Vararg` version (because of `AbstractArray{<:Number}` typing)